src/Symfony/Component/Validator/Tests/Mapping/Loader/PropertyInfoLoaderTest.php

```diff
@@ -46,6 +46,7 @@ public function testLoadClassMetadata()
                 'alreadyMappedNotBlank',
                 'alreadyPartiallyMappedCollection',
                 'readOnly',
+                'nonExistentField',
```

Comment:
No asserts were added, but, without the fix, these changes are enough to trigger an exception to be thrown while running this test.